### PR TITLE
ava test runner

### PR DESCRIPTION
**Pull request by osdiab**
easier to read output and no need for `t.end()`

no big deal but saw the earlier conversation re tape, and using it is super easy to implement.

Screen Shot 2020-05-20 at 15 39 39

### CODE REVIEW

**Review by lo1tuma — APPROVED** [review]
LGTM

**Comment by osdiab** [comment]
fixed the build - this, #20 , and #23 all fix #21 but as such will also merge conflict so whichever one goes first, i can resolve those conflicts afterward

File: package.json [hunk 1]
```diff
@@ -20,10 +20,20 @@
...
+  "ava": {
+    "extensions": [
+      "ts"
+    ],
+    "files": [
+      "tests/**/*.test.ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ]
```
Comment (gillchristian) [inline package.json]: [Q] Does AVA support somthing like `ava.config.json`? I'd rather not pollute the package.json. Although that would me more files in the root but 🤷‍♂️

**Review by gillchristian — APPROVED** [review]
LGTM!

File: package.json (hunk 1 above)
Comment (osdiab) [reply package.json]: It does, can switch it to that!

**Comment by osdiab** [comment]
Ready to go!

**Comment by gillchristian** [comment]
@osdiab same here :)

**Comment by osdiab** [comment]
cool that should do it!

**Comment by gillchristian** [comment]
Awesome! Thanks for all the PRs 💪 👏 🎉

**Pull request merged**